Scripts/Haptics/XRNodeHapticPulse.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace VRTK.Core.Haptics
+{
+    using UnityEngine;
+    using UnityEngine.XR;
+
+    /// <summary>
+    /// Creates a timed haptic pulse on a <see cref="XRNode"/> supported haptic device.
```

Comment:
```suggestion
    /// Creates a timed haptic pulse on a <see cref="XRNode"/>.
```
A "device" is a different thing in the UnityEngine.XR namespace, thus we should only talk about "nodes" here to prevent confusion.